server/routes/api/articles.js

```diff
@@ -11,6 +11,7 @@ const router = express.Router();
 
 router.post('/articles', checkToken, validate(articleSchema.create), Articles.create);
 router.get('/articles', Pagination.validatePagination, Articles.getAll);
+router.get('/articles/private', Pagination.validatePagination, checkToken, Articles.getPrivateArticles);
```

Comment:
@gracelungu, wouldn't it be good to not protect this route and add the username in the URL as a parameter so that it can be easy to access it outside of the user's profile?